test/game.test.js

```diff
@@ -158,17 +155,31 @@ describe("Game", () => {
 		});
 
 		test("displays the character selected", () => {
-			const expectedCharacter = `Character Selected: ${game.transientData["character-select"].choice.title}`;
-			expect(game.add.text).toHaveBeenCalledWith(150, 200, expectedCharacter, {
+			const characterTitle = "Kawabashi";
+			const expected = `Character Selected: ${characterTitle}`;
+			let mockGetUnique = jest.fn().mockImplementation(() => ({ title: characterTitle }));
+			collectionsModule.collections = {
+				get: jest.fn(() => ({ get: jest.fn(), set: jest.fn(), getUnique: mockGetUnique })),
+			};
+			game.create();
+
+			expect(game.add.text).toHaveBeenCalledWith(150, 200, expected, {
 				font: "32px ReithSans",
 				fill: "#f6931e",
 				align: "center",
 			});
 		});
 
 		test("displays the level selected", () => {
-			const expectedLevel = `Level Selected: ${game.transientData["level-select"].choice.title}`;
-			expect(game.add.text).toHaveBeenCalledWith(150, 250, expectedLevel, {
+			const levelTitle = "Hard level";
+			const expected = `Level Selected: ${levelTitle}`;
+			let mockGetUnique = jest.fn().mockImplementation(() => ({ title: levelTitle }));
```

Comment:
I think mockImplenentation can just be replaced these days by passing it into the functione.g:

`jest.fn(() => ({ title: levelTitle }));`